dev_utils/run_integration_test.sh

```diff
@@ -20,18 +20,18 @@ if [ "$1" != "" ]; then
     FILE="$(realpath "$1")"
 fi
 
-file=$(basename "$FILE")
+file=test/$(basename "$FILE")
 
 cd "$(dirname "$0")" || exit
 
-s3cmd -c s3cmd.conf put "$FILE" s3://inbox/test/"$file"
+s3cmd -c s3cmd.conf put "$FILE" s3://inbox/"$file"
```

Comment:
Any particular reason for this change?

Reply:
Without the change there is a mismatch between the file upload location and the _filepath_ in the mq messages. This resulted in this error:
ingest     | time=“2021-09-15T14:07:46Z” level=info msg=“Received work (corr-id: 1, filepath: dummy_data.c4gh, user: test)”
ingest     | time=“2021-09-15T14:09:46Z” level=error msg=“NoSuchKey: The specified key does not exist.\n\tstatus code: 404, request id: 16A5042243A3F096, host id: ”
ingest     | time=“2021-09-15T14:09:46Z” level=error msg=“Failed to open file to ingest (corr-id: 1, user: test, filepath: dummy_data.c4gh, reason: NoSuchKey: The specified key does not exist.\n\tstatus code: 404, request id: 16A5042243A3F096, host id: )”